src/chunk_append/exec.c

```diff
@@ -331,7 +364,7 @@ chunk_append_exec(CustomScanState *node)
 		 * more subplans, return the empty slot set up for us by
 		 * chunk_append_begin.
 		 */
-		choose_next_subplan(state);
+		state->choose_next_subplan(state);
```

Comment:
is the comment above this and below this out of date now? We don't return the empty slot here right?